client/html/templates/catalog/filter/search-body-default.php

```diff
@@ -105,6 +105,11 @@
 <section class="catalog-filter-search">
 	<h2><?php echo $enc->html( $this->translate( 'client', 'Search' ), $enc::TRUST ); ?></h2>
 	<input class="value" type="text" name="<?php echo $name; ?>" value="<?php echo $phrase; ?>" data-url="<?php echo $suggestUrl; ?>" data-hint="<?php echo $hint; ?>" /><!--
+    Comment end to prevent merge conflict and thus non-fast forward merge. -->
+	<div style="display: <?php if (empty($phrase)) echo 'none'; else echo 'inline-block'?>;" class="pp_default">
```

Comment:
You shouldn't use inline CSS if not absolutely necessary. CSS selectors are quite powerful and something like this should work too:
```
.catalog-filter-search .value[value=''] + .pp_default {
    display: none;
}
.catalog-filter-search:not(.value[value='']) .value + .pp_default {
    display: inline-block;
}
```
Looks a bit complicated but can be changed in themes without the need to change the templates